src/Umbraco.Core/EmbeddedResources/Lang/en.xml

```diff
@@ -428,6 +428,8 @@
     <key alias="compilationDebugCheckErrorMessage">Debug compilation mode is currently enabled. It is recommended to
       disable this setting before go live.
     </key>
+    <key alias="runtimeModeCheckSuccessMessage">Runtime mode is set to production.</key>
+    <key alias="runtimeModeCheckErrorMessage">Runtime mode is not set to production. It is recommended to change this setting to Production before go live.</key>
```

Comment:
I don't think this makes a lot of sense, as the runtime mode is environment specific rather than being something you "turn on" at some point (unlike Debug mode, which one might enable temporarily on a production server).

How about this?
```suggestion
    <key alias="runtimeModeCheckErrorMessage">Runtime mode is not set to Production. It is recommended to set the Runtime Mode to Production for live/production environments.</key>
```